README.md

```diff
@@ -1,3 +1,6 @@
+# Archived
+
+This project has been archived. Its contents have been merged into [Microsoft/hcsshim](https://github.com/microsoft/hcsshim) where development will continue. Some code organization has changed as part of the merge, please see [the merge PR](https://github.com/microsoft/hcsshim/pull/973) for details on where things have moved. Any issues regarding opengcs should be opened on the hcsshim repo.

```

Comment:
@dcantah if you have details on the code organization changes I can include here, please let me know.

Reply:
```suggestion
This project has been archived. Its contents have been merged into [Microsoft/hcsshim](https://github.com/microsoft/hcsshim) where development will continue. Some code organization has changed as part of the merge, please see below for details on where things have moved. Any issues regarding opengcs should be opened on the hcsshim repo.

Changes:
* Everything in /internal has been moved to [github.com/Microsoft/hcsshim/internal/guest](https://github.com/microsoft/hcsshim/tree/master/internal/guest) besides a few redundant packages that had equivalents in hcsshim. These include /internal/debug, /internal/oc and /internal/log. These are now located at github.com/Microsoft/hcsshim/internal/debug, github.com/Microsoft/hcsshim/internal/oc, and github.com/Microsoft/hcsshim/internal/log.
* /service/libs/commonutils has moved to github.com/Microsoft/hcsshim/internal/guest.
* Any non Go code lives at the root of github.com/Microsoft/hcsshim now, just as it did in this repository. This includes /vsockexec, /init, and /hack.
* The main gcs and gcstools binaries are now built and located at github.com/Microsoft/hcsshim/cmd/gcs and  github.com/Microsoft/hcsshim/cmd/gcstools. 
* Makefile lives top level at github.com/Microsoft/hcsshim/Makefile and same commands will work to build everything.
```

Reply:
Thanks @dcantah, looks good!